### ISSUE TITLE
Add <Svenska> [sv]

### ISSUE THREAD

**Issue by ignotus666** [jamulussoftware/jamulussoftware.github.io#999]
Add Swedish language files.

Better wait until Weblate is up and running again - then add the "new language" label to trigger it.

**Comment by ann0see** [comment]
Most likely you’ll need to update permissions (since the workflow failed)

**Comment by ignotus666** [comment]
> since the workflow failed

What workflow?

Edit: never mind, just saw it.

**Comment by ann0see** [comment]
https://github.com/jamulussoftware/jamuluswebsite/actions/runs/10126433912

But I have the feeling that release is just severely outdated. And we whitelist actions on next-release. Most of the updates to gh actions went to next-release and the branches somewhat diverged.

**Comment by ignotus666** [comment]
Yes, I just added the (deprecated) action to the whitelist but ideally we should merge the updated workflows from `next-release` into `release`.

**Comment by ignotus666** [comment]
@ann0see can we unlock Weblate now?

**Comment by ann0see** [comment]
There may still be documentation changes outstanding? Screenshots? Before we can start the translation run.

**Comment by pljones** [comment]
Is this targetted at 3.11.0?  If so, please add the release.

**Comment by ignotus666** [comment]
With Weblate I see website translations as more of a rolling situation - they trickle in and then maybe ramp up when we put out a call as a release approaches, but many of them never make it to complete. The volume is just way higher than the app. So I think targeting a specific release is a bit meaningless.

**Comment by pljones** [comment]
Do we then need to update the release process documentation so that we no longer worry about the website being up to date with the application?  That doesn't seem right to me.

I'd expect any change to have a reason and the delivery of that change to be targeted -- if it's not dependent upon the application, then okay, maybe it's not specific to a release but it should still be targeted to the current release if that's when it's expected to be done.  It can always get moved.

**Comment by ann0see** [comment]
For new languages probably we could get away from targeting releases, but I‘d suggest that we still do translation sprints.

**Comment by ignotus666** [comment]
> Do we then need to update the release process documentation so that we no longer worry about the website being up to date with the application? That doesn't seem right to me.

Of course the website should be up to date with the app - at least the default, English version. But given the volume of text, and more so for a new language, tying translations to a specific release is wishful thinking. If it happens, great, but only a handful of languages are maintained at a complete (or near enough) level. But I still agree that we should do translation sprints - the proximity of a release is a motivator.

**Comment by pljones** [comment]
OK, I'd suggest we don't _start_ new languages proximally to a release as they cut across the release cycle, if we're going to to allow them not to tie in.

**Comment by ignotus666** [comment]
I would argue that as they're not necessarily tied to a release, it would make sense to add SV now. If we don't, the translator won't see any of their translation work on the website until the next release (and won't get to start on it now; @genesisproject2020 showed some interest in doing it - maybe he could weigh in), which could be quite far away. As I said, the volume of text is very large and motivation is a big factor.

**Comment by ann0see** [comment]
Yes. Let's add it now.

**Comment by ann0see** [comment]
Check the regex in the automation.

https://regexr.com/ can serve as nice way to debug.

**Comment by ignotus666** [comment]
I can't see what's wrong with it - I use [this site](https://www.regextester.com/) and it checks out. I could also swear I tested adding a language on my repo and it worked.

**Comment by ann0see** [comment]
I think I know what the problem is. It's taking the tooling from release not next release.

**Comment by ignotus666** [comment]
Yes, but that should be fixed - the scripts were updated on release.

**Comment by ignotus666** [comment]
I think I've got it - there are missing quotes in this line:

`if [[ $LANG_CODE =~ ^[a-z]{2}(-[A-Z]{2})?$ && $ISSUE_TITLE =~ <[^>]+> ]]; then `

It should be:

`if [[ "$LANG_CODE" =~ ^[a-z]{2}(-[A-Z]{2})?$ && "$ISSUE_TITLE" =~ <[^>]+> ]]; then`

**Comment by ignotus666** [comment]
Should we commit directly to release or do it via a PR?

**Comment by ann0see** [comment]
PR

**Comment by ann0see** [comment]
Please check the regex. Now it's a syntax error.

**Comment by ignotus666** [comment]
Yeah, I'm doing tests on my repo and will make a new PR when it's fixed.

**Comment by ignotus666** [comment]
It seems that `sed` doesn't like `<>` characters in the regex and gets confused. Fixed by using `grep` instead. I also split the errors so it tells you specifically what part of the issue title is wrong. Tested on my repo and is working.

**Comment by ignotus666** [comment]
Urgh... Weblate only updated with the "startpage" (1-Index) file for Swedish. I'm looking at the settings and they seem ok. I don't know what could have changed and adding languages worked fine in the past. @ann0see any ideas?

**Comment by ann0see** [comment]
Yes. You need to force rescan the po files in the "Danger section" I suppose. Had a comparable issue yesterday.

**Comment by ignotus666** [comment]
Ah, I was doing "update" and that wasn't working. Fixed. BTW, I changed the lang code from sv to sv-SE to keep it more consistent with the app (#1022). I know other languages should probably have the same done with them (es, de...). The startpage has already been translated I see.
